test/README.md

```diff
@@ -1,5 +1,63 @@
 ## End-to-End Testing
 
+### AWS E2E Testing
+Featured in this directory is a `./awsE2ESolutionSetup` script whose goal is to encapsulate deploying an E2E environment in AWS. This script will perform the following key actions:
+1. Deploy an Elasticsearch source cluster to EC2 using the following [CDK](https://github.com/lewijacn/opensearch-cluster-cdk/tree/migration-es)
+   * The CDK context options for this CDK can be used to customize this deployment and can be provided with a custom context file using the `--context-file` option
+   * A sample CDK context file can be seen [here](defaultCDKContext.json)
+2. Build the Docker Images
+3. Deploy the Migration Assistant CDK (containing the Migration tooling and potentially an OpenSearch Domain) located in this repo [here](../deployment/cdk/opensearch-service-migration)
+   * The CDK context options for this CDK can be used to customize this deployment and can be provided with a custom context file using the `--context-file` option
+   * A sample CDK context file can be seen [here](defaultCDKContext.json)
+4. Add the traffic stream source security group created by the Migration Assistant CDK to the source cluster load balancer
+5. Run the `./startCaptureProxy.sh` script for starting the Capture Proxy process on each source cluster node/instance 
+   * This script is automatically added to each EC2 instance on creation
+
+#### Configuring a Historical Migration focused E2E environment
+A more focused experience for only testing historical migrations can be achieved with this script by customizing the CDK context options to minimize live capture resources like so:
+```
+{
+  "rfs-backfill": {
+    "stage": "<STAGE>",
+    "vpcId": "<VPC_ID>",
+    "engineVersion": "OS_2.11",
+    "domainName": "os-cluster-<STAGE>",
+    "dataNodeCount": 2,
+    "openAccessPolicyEnabled": true,
+    "domainRemovalPolicy": "DESTROY",
+    "artifactBucketRemovalPolicy": "DESTROY",
+    "kafkaBrokerServiceEnabled": true,
+    "trafficReplayerServiceEnabled": false,
+    "reindexFromSnapshotServiceEnabled": true,
+    "sourceClusterEndpoint": "<SOURCE_CLUSTER_ENDPOINT>"
```

Comment:
Can we get a dictionary or breakdown of all the magic "phrases" that we can embed in these context files, and what they do?  We should probably explain how we're auto-replacing them with concrete values, too.

Reply:
Yes meant to add this, have included now